### DISCUSSION TITLE
Jupyter notebook for examples

### DISCUSSION THREAD

**Discussion by ghi94116** [fermulerpy/fermulerpy#22] [Ideas]
Can we add jupyter notebooks for showcasing the usefulness of this package

**Comment by MAG-BOSS** [comment]
Yes , go ahead !
This would be good.

**Comment by ghi94116** [comment]
Thanks! I am working on it.

**Comment by MAG-BOSS** [comment]
#23

**Comment by MAG-BOSS** [comment]
You were talking about adding jupyter notebooks in docs section??
